pkgs/by-name/un/undertime/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, python3Packages
+, fetchFromGitLab
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "undertime";
+  version = "4.0.0";
+  pyproject = true;
+
+  src = fetchFromGitLab {
+    owner = "anarcat";
+    repo = pname;
+    rev = version;
+    hash = "sha256-BshgSnYaeX01KQ1fggB+yXEfg3Trhpcf/k4AmBDPxy8=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    dateparser
+    ephem
+    pytz
+    pyyaml
+    termcolor
+    tabulate
+    tzlocal
+  ];
+
+  meta = {
+    changelog = "https://gitlab.com/anarcat/undertime/-/raw/${version}/debian/changelog";
+    description = "pick a meeting time across timezones from the commandline";
```

🔠 Would be nice to start the description with an uppercase letter :)

```suggestion
    description = "Pick a meeting time across timezones from the commandline";
```